wled00/FX_fcn.cpp

```diff
@@ -1438,6 +1453,8 @@ uint16_t WS2812FX::getFps() const {
 void WS2812FX::setTargetFps(uint8_t fps) {
   if (fps > 0 && fps <= 120) _targetFps = fps;
   _frametime = 1000 / _targetFps;
+  if (_frametime < 1) _frametime = 1;           // better safe than sorry
```

Comment:
These two lines are completely unnecessary. `_targetFps` can never be less than 1 or more than 120. The first line makes sure of that.
Hence `_frametime` can only be between 1000 and 8 respectively (usually 23 (1000/42)), which is wrong as it is *more* than that due to integer math. Perhaps adding 1 to `_frametime` is a better option.

Reply:
I'll remove line 1456. The line below that is actually necessary for FPS_UNLIMITED test mode.

Reply:
check for (_frametime < 1) removed.